LuckyPills/Plugin.cs

```diff
@@ -9,23 +9,23 @@ public class Plugin : Plugin<Config>
         public override string Author { get; } = "Reddking improved by Parkeymon";
         public override string Name { get; } = "Lucky Pills";
         public override Version Version { get; } = new Version(2, 1, 2);
-        public override Version RequiredExiledVersion { get; } = new Version(2, 11, 1, 0);
+        public override Version RequiredExiledVersion { get; } = new Version(3, 0, 0, 0);
```

Comment:
set the required to what you actually use there were breaking changes between 3.0.0 and latest pre-release

Reply:
The latest EXILED version is listed at 3.0.0.0, no?

Reply:
it doesnt load the plugin if its not 3.0.0.0